shared/src/formationData.ts

```diff
@@ -139,7 +139,7 @@ export interface FormationFormData extends FormationAddress {
   readonly agentEmail: string;
   readonly agentOfficeAddressLine1: string;
   readonly agentOfficeAddressLine2: string;
-  readonly agentOfficeAddressMunicipality: Municipality | undefined;
+  readonly agentOfficeAddressMunicipality: string;
   readonly agentOfficeAddressCity: string | undefined;
```

Comment:
What's the difference between these 2 fields?

Reply:
Municipality is an object, we are not replacing it with a string

Reply:
@Faraz Akhtar, just verifying that you meant **now** _NOT_ **not** in this comment.

Reply:
I deleted it, JP's question is valid